pkgs/by-name/bu/buttermanager/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, wrapGAppsHook
+, qt5
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "buttermanager";
+  version = "2.5.1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "egara";
+    repo = "buttermanager";
+    rev = version;
+    hash = "sha256-MLYJt7OMYlTFk8FCAlZJ1RGlFFXKfeAthWGp4JN+PfY=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    pyqt5
+    pyyaml
+    sip
+    tkinter
+    setuptools
+  ];
+
+  nativeBuildInputs = [ wrapGAppsHook qt5.wrapQtAppsHook ];
+
+  qtWrapperArgs = [
+    "--set QT_QPA_PLATFORM_PLUGIN_PATH ${qt5.qtbase.bin}/lib/qt-${qt5.qtbase.version}/plugins/platforms"
+  ];
```

Comment:
I tested on Arch and didn't seem to need this. Can you double-check if you still need this on your end?

Reply:
Yeah works for me on NixOS + xmonad without it as well now, maybe version bump in the middle of this PR fixed it

Reply:
I'd guess it had more to do using `nativeBuildInputs` as the necessary hooks wouldn't be run otherwise.